config/config.go

```diff
@@ -72,6 +72,11 @@ func (c Config) Validate() error {
 				return fmt.Errorf("Missing room in HipChat config: %s", proto.MarshalTextString(hcc))
 			}
 		}
+		for _, sc := range nc.SlackConfig {
+			if sc.WebhookUrl == nil {
+				return fmt.Errorf("Missing webhook url in Slack config: %s", proto.MarshalTextString(sc))
```

Comment:
Oh yes, I hadn't even looked that far yet. While this was about the word "url" in the error message (also error-messages should all start out lower-cased, but the others here are all wrong too, so maybe keep yours starting upper-cased for now), the general rule is also to avoid "initialisms" in variable names: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

So `contentTypeJson` -> `contentTypeJSON`.

The exception are protobuf-generated names like the `WebhookUrl`. You can't use `WebhookURL` there because the Go name is autogenerated from the protocol buffer schema definition.